### ISSUE TITLE
I can't view analytics

### ISSUE THREAD

**Issue by klaksik** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#426]
I can't view analytics
![image](https://user-images.githubusercontent.com/89664351/154625358-591d7377-d56d-4280-8f49-18236cb2eaa5.png)

**Comment by byaura** [comment]
Did you add this to your run.py file? 
`from TwitchChannelPointsMiner import TwitchChannelPointsMiner
twitch_miner = TwitchChannelPointsMiner("your-twitch-username")
twitch_miner.analytics(host="127.0.0.1", port=5000, refresh=5)   # Analytics web-server
twitch_miner.mine(followers=True, blacklist=["user1", "user2"])`

**Comment by Appendme** [comment]
I Perhaps I have the same problem. I started the miner in a docker container and when I try to connect, it gives an error: "Recv failure: Connection reset by peer", but if I try to connect inside the container, it happens without errors.

Apparently, connections from another network are blocked, I encountered something similar when setting up a proxy in nginx, and there it is fixed like adding "proxy_set_header Host localhost;", but what to do here?
